generic/tests/netperf.py

```diff
@@ -106,6 +105,37 @@ def env_setup(session, ip, user, port, password):
         remote.scp_to_remote(ip, shell_port, username, password,
                              agent_path, "/tmp")
 
+    def mtu_set(mtu):
+        """
+        Set server/client/host's mtu
+
+        :param mtu: mtu value to be set
+        """
+
+        linux_mtu_cmd = params.get("linux_mtu_cmd")
+        windows_mtu_cmd = params.get("windows_mtu_cmd")
+        error_context.context("Changing the MTU of guest", logging.info)
+        if params.get("os_type") == "linux":
+            ethname = utils_net.get_linux_ifname(server_ctl, mac)
+            ssh_cmd(server_ctl, linux_mtu_cmd % (ethname, mtu))
```

Comment:
It is better to be `server_mtu_cmd`, `client_mtu_cmd` and `bridge_mtu_cmd`.

Reply:
the server can be linux or windows, so the mtu command is not same. it's not suitable with server_mtu_cmd. clinet and bridge should be linux system, the command are the same.

Reply:
The following approach could fit your requirement.
```
server_mtu_cmd = ${linux_cmd}
Windows:
    server_mtu_cmd = ${windows_cmd}
```

And it won't be an issue for future extension.

Reply:
List four params , server_mtu_cmd (two), client_mtu_cmd,bridge_mtu_cmd, and three of them are using the same linux command, what's the advantage with them?

Reply:
To ensure there is no potential bug which is introduced by your change.

Reply:
I have update , please check it